crates/types/src/joints.rs

```diff
@@ -51,6 +55,17 @@ impl Mul<f32> for HeadJoints {
     }
 }
 
+impl Div<HeadJoints> for HeadJoints {
+    type Output = HeadJoints;
+
+    fn div(self, right: HeadJoints) -> Self::Output {
```

Comment:
Wieso right? Hätte eher sowas wie other gewählt. Gibt es da eine convention?
Gilt natürlich auch für alle anderen Stellen

Reply:
Yes, operands of binary operators are usually called left-hand side (lhs) and right-hand side (rhs).
This makes the order more explicit compared to using "other", which is especially important for non commutative operations such as division.

Reply:
Yes, prefer `other` or `left` or `right`